hvac/api/secrets_engines/kv_v2.py

```diff
@@ -283,7 +283,7 @@ def delete_secret_versions(self, path, versions, mount_point=DEFAULT_MOUNT_POINT
         :type path: str | unicode
         :param versions: The versions to be deleted. The versioned data will not be deleted, but it will no longer be
             returned in normal get requests.
-        :type versions: int
+        :type versions: list of int
```

Comment:
I believe the correct format would be like this
```suggestion
        :type versions: list[int]
```

Reply:
OK, I thought you type hint it in such a way. I replaced it to 'list[int]'